src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -96,7 +96,7 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <PackageTags>Microsoft;Azure;WebJobs;Durable;Extension;Orchestration;Workflow;Functions</PackageTags>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=2028464</PackageLicenseUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/Azure/azure-functions-durable-extension/7e01577f57fdfdb15d20fa66b09c3714926e0aea/LICENSE</PackageLicenseUrl>
```

Comment:
FWIW, this property has been deprecated for a while. The recommendation is to use `<PackageLicenseExpression>` instead. Not sure how much we want to take on for this release, but here is an example of how the SQL backend project does this: https://github.com/microsoft/durabletask-mssql/blob/4ff7b2f5364aadf0b8d322a4eeb86df153ff746e/src/common.props#L31

Reply:
Honestly, that seems simpler than my fix, and since it will get rid of the nuget.org warning as well, I'll go with that.